protocol/contracts/core/RebasingERC20.sol

```diff
@@ -46,6 +46,43 @@ abstract contract RebasingERC20 is ERC20 {
         return toTokenAmount(shareBalanceOf[account]);
     }
 
+    /**
```

Comment:
OZ recommends override the `_update()` function for all of these, which imo is better than introducing new non-virtual functions below